topbeat/etc/fields.yml

```diff
@@ -8,6 +8,90 @@
         Set to "system" to indicate that the statistics are system-wide.
       required: true
 
+- key: core
+  title: "CPU statistics per core"
+  description: >
+    Contains CPU statistics for each core. These statistics are the details that you can get by running the *top* command on Unix systems.
+  fields:
+    - name: id
+      type: long
+      description: >
+       The id of the core X.
+
+    - name: user
+      type: long
+      description: >
+       The amount of CPU time spent in user space on core X.
```

Comment:
see `man sysconf` and search for ticks. Default value for linux is 100, but do not rely on it. Without knowing the frequency the tick counter is incremented, ticks is quite useless. Better would be some measure in seconds or milliseconds.